HowTos/firewall_network_faq.rst

```diff
@@ -13,7 +13,7 @@ What is Aviatrix Firewall Network (FireNet)?
 Aviatrix Firewall Network (FireNet) is a turn key network solution to deploy firewall instances in the 
 cloud, as shown in the diagram below. 
 
-|firewall_network|
+<img width="1274" alt="image" src="https://user-images.githubusercontent.com/69173400/89824008-44939c80-db07-11ea-9971-7fd7490d6088.png">
```

Comment:
I'm not sure if the image can be pulled from your repo. Please copy the png file to the local directory like the other ones.